docs/source/github.rst

```diff
@@ -28,7 +39,7 @@ Now that you have forked the repository, you will clone it locally on your compu
 
 - Go to the forked GitHub directory webpage. It should be something like: ::
 
-https://github.com/your_username/EasyHPC/
+	https://github.com/your_username/EasyHPC/
```

Comment:
Check which is the right tabulation for code in .rst files. I don't know how many spaces must be added after `::`